tests/printer/typexpr/__snapshots__/render.spec.js.snap

```diff
@@ -327,7 +327,8 @@ external test: (
   foo,
   bar,
   baz,
-) => @attr {..
+) => @attr
```

Comment:
The line break is shady.

Reply:
I think before that, it went into a branch where it saw an attribute right before a type constructor (`Js.t`), so formatted that `Js.t` on the same line as the attribute, which then triggered the type constructor + object `<{` hugging branch.

But I've already discovered an existing error! `@foo {"bar": baz}` printing removes the `@foo`.